### ISSUE TITLE
[Bug report] @babel/core@7.0.0-beta.47 may have some problems.

### ISSUE THREAD

**Issue by tyrion-yu** [vuejs/vuepress#601]
#### Version

0.10.1

#### Steps to reproduce

add `.vuepress/theme/Layout.vue`

``` vue
<template>
    <Layout>
        <div class="secrecy" slot="page-bottom">
            My footer.
        </div>
    </Layout>
</template>

<script>
  import Layout from '@default-theme/Layout.vue';

  export default {
    components: {
      Layout,
    },
  };
</script>
```

`package.json`

``` json
"devDependencies": {
    "babel-core": "^6.26.3",
    "babel-loader": "^7.1.4",
    "babel-preset-env": "^1.7.0",
}
```

add  `.babelrc` to your root dir

``` json
{
    "presets": ["env"]
}
```

run `vuepress dev` then:

```
WARNING in ./docs/.vuepress/theme/Layout.vue?vue&type=script&lang=js 1:276-279
"export 'default' (imported as 'mod') was not found in '-!../../../node_modules/cache-loader/dist/cjs.js??ref--3-0!../../../node_modules/vuepress/node_modules/babel-loader/lib/index.js??ref--3-1!../../../node_modules/vue-loader/lib/index.js??vue-loader-options!./Layout.vue?vue&type=script&lang=js'
 @ ./docs/.vuepress/theme/Layout.vue
 @ ./node_modules/vuepress/lib/app/.temp/routes.js
 @ ./node_modules/vuepress/lib/app/app.js
 @ ./node_modules/vuepress/lib/app/clientEntry.js
 @ multi ./node_modules/vuepress/lib/app/clientEntry.js
```

The VuePress Site Broken.

#### What is expected?

remove the file `.babelrc`, then everything is FINE.

#### What is actually happening?

The VuePress use `babel@^7.0.0-beta.47` which is incompatible with the project using `babel@^6.x`.

#### Other relevant information

- Your OS: macOS Sierra 10.12.6
- Node.js version: v8.9.4
- Browser version: chrome@latest
- Is this a global or local install? Both are not Okey.
- Which package manager did you use for the install? yarn

**Comment by Genert** [comment]
+1 here. Any ideas?

**Comment by mattixittam** [comment]
I have a related issue, I think. My base project uses babel 6.x and vuepress chokes on the babel-preset-env (1.6.1) in the base project:

```
Module build failed (from ./node_modules/vuepress/node_modules/babel-loader/lib/index.js):
Error: Plugin/Preset files are not allowed to export objects, only functions.
In /Users/myusername/CODE/myprojectname/node_modules/babel-preset-stage-2/lib/index.js
```

**Comment by mattixittam** [comment]
+1, babel 7 is choking on my base project babel 6 .babelrc and presets

**Comment by ulivz** [comment]
It should be fixed at c5002010581391478aa5ca2d36f6e6c6dff9bbb9.
